.github/workflows/master.yml

```
@@ -26,8 +26,18 @@ jobs:
       - run: npm ci
       - run: npm run test
 
-  publish:
-    needs: [test-8, test-10]
+  e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+      with:
+        node-version: 10
+      - run: npm ci
+      - run: npm run e2e
+
+  publish:s
+    needs: [test-8, test-10, test-12]
```

change to `e2e` here as we don't want publish moving forward if `e2e` fails

```suggestion
    needs: [test-8, test-10, e2e]
```